modules/deleting-machine-pools.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa_nodes/rosa-managing-worker-nodes.adoc
+
+:_content-type: CONCEPT
+[id="deleting-machine-pools{context}"]
+= Deleting a machine pool
+
+You can delete a machine pool in the event that your workload requirements have changed and your current machine pools no longer meet your needs.
+// Over time, users may find that their workload needs have changed, and may want to modify the various machine pool settings. While many of these settings can be modified, certain settings (for example, instance types and availability zones) cannot be changed once a machine pool is created. If a user finds that these settings are no longer meeting their needs, they can delete the machine pool in question and create a new machine pool with the desired settings.
+You can
+delete machine pools
+using the
+ifdef::openshift-rosa[]
+Openshift Cluster Manager or the ROSA CLI (`rosa`).
+endif::openshift-rosa[]
+ifndef::openshift-rosa[]
+Openshift Cluster Manager.
+endif::[]
+
+// Users that wish to delete the default machine pool that is automatically created during the installation of a {product-title} (ROSA) cluster can do so using the OCM or ROSA CLI.
+//
+[NOTE]
+====
+In order to  delete a machine pool, there must be at least one other machine pool that does not include any taints and has at least two nodes (in the case of a single-AZ cluster) or three nodes (in the case of a multi-AZ cluster).
+====
+ifndef::openshift-rosa[]
+.Prerequisites
+
+ifdef::openshift-rosa[]
+* You created a ROSA cluster.
+endif::openshift-rosa[]
+//* The newly created cluster is in the ready state.
+ifndef::openshift-rosa[]
+* You have created an {product-title} cluster.
+* The newly created cluster is in the ready state.
+endif::[]
+
+* You have an existing machine pool without any taints and with at least two instances (for a  single-AZ cluster) or three instances (for a multi-AZ cluster).
+
+.Procedure
+. From {cluster-manager-url}, navigate to the *Clusters* page and select the cluster that contains the machine pool that you want to delete.
+
+. On the selected cluster, select the *Machine pools* tab.
+
+. Under the *Machine pools* tab, click the options menu {kebab} for the machine pool that you want to delete.
+. Click Delete.
+
+The selected machine pool is now deleted.
+endif::[]
```

Comment:
I do not think you need this closing argument. You have closed both of your conditions above.

Reply:
You are correct @mletalie. I had orginally missed the condition you set at line 24.